products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp

```diff
@@ -51,7 +51,12 @@ namespace app
         });
 
         addActionReceiver(app::manager::actions::DisplayLowBatteryScreen, [this](auto &&data) {
-            handleLowBatteryNotification(std::move(data));
+            /**
```

Comment:
Where is now low battery screen processed?

Reply:
Switch to low battery screen is handled in `SystemBrownout` action receiver.

Reply:
If you meant low battery notification (when real SOC is between 5-10%) then it was dropped after discussion with @mjan-dev and analyzing flow from Figma.